src/oscar/apps/dashboard/permissions.py

```diff
@@ -140,6 +140,23 @@ class DashboardPermission:
             "voucher.view_voucher",
             "voucher.delete_voucher",
         ],
+        # Granular dashboard permissions
+        "view-order": [
+            "is_staff",
+            "order.view_order",
+        ],
+        "edit-order": [
+            "is_staff",
+            "order.change_order",
+        ],
+        "view-product": [
+            "is_staff",
+            "catalogue.view_product",
+        ],
+        "view-stock": [
```

Comment:
This should be updated to view-stock-alert. `partner.view_stockalert`. is_staff seems redundant. Please add a default permission attribute and then add merge it in the get method.